### PR TITLE
Add translation to custom fields

### PR DESCRIPTION
**Pull request by Kagemaru**

### CODE REVIEW

File: lib/decidim/decidim_awesome/custom_fields.rb [hunk 1]
```diff
@@ -60,6 +64,27 @@ def apply_to_first_textarea
         textarea["userData"] = [xml]
         textarea["name"]
       end
+
+      def translate_values!
+        deep_transform_values!(@fields) do |value|
+          next value unless value.is_a? String
+          next value unless (match = value.match(/^(.*\..*)$/))
```
Comment (microstudi) [inline lib/decidim/decidim_awesome/custom_fields.rb:71]:
```suggestion
          next value unless (match = value.match(/^(decidim\..*)$/))
```

getting back to your idea of marking the if you want to translated or no, maybe we could just force to start with `decidim.` any thing that needs to be translated? what you think?

Comment (Kagemaru) [reply lib/decidim/decidim_awesome/custom_fields.rb:71]: Hmm... that would work. How about another prefix that isn't as loaded, like `custom_fields`?

Comment (microstudi) [reply lib/decidim/decidim_awesome/custom_fields.rb:71]: yes, however,... do we want to distinct between translation keys or just any direct text? can't we just write the key if no translation is available? I mean, ActiveRecord works just this way. 
I we do this approach we leave to the user how "tidy" they want to be.

I am not concerned about people using a key as a final text, it seems to me that this is a very improbable case, and even in this case you can simply create the key.

Comment (microstudi) [reply lib/decidim/decidim_awesome/custom_fields.rb:71]: I am insisting in this way to do thinkg so we don't have to force or explain how to deal with it, it "just works"

Comment (microstudi) [reply lib/decidim/decidim_awesome/custom_fields.rb:71]: And, finally, we could use `decidim.` as a default and add a configuration option to let the developer decide a different thing if they want (even if they want to leave it empty). So everybody happy

Comment (Kagemaru) [reply lib/decidim/decidim_awesome/custom_fields.rb:71]: Ah, I'm ok with not using a marker at all... That's how I've handled it now and I like it.

**Review by microstudi — APPROVED** [review]

**Pull request merged**